ci-operator/step-registry/baremetalds/assisted/setup/baremetalds-assisted-setup-commands.sh

```diff
@@ -81,6 +81,7 @@ if [ "${JOB_TYPE:-}" = "presubmit" ] && (( ! \${IS_REHEARSAL} )); then
   echo "export OPENSHIFT_VERSION=4.7" >> /root/config
 
   if [ "${REPO_NAME:-}" = "assisted-service" ]; then
+    echo "export SERVICE_BASE_REF=${PULL_BASE_REF:-master}" >> /root/config
```

Comment:
we need something like this for assisted-installer as well?

Reply:
assisted-installer is used only for day2 operations. So we can do that, but it's actually not supported via the CI. 
So once we'll have such job we can add that